TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java

```diff
@@ -163,33 +163,33 @@ protected Void doInBackground(Void... params) {
 
                                 JSONObject innerObject = array.getJSONObject(jIndex);
 
-                                String id = innerObject.getString("ID");
-                                String name = innerObject.getString("NAME");
+                                String id = innerObject.getString(getString(R.string.ID));
```

Comment:
Here, it is `ID` and `NAME` but you changed them to `ID:` and `NAME:` so fetching of data does not occur

Reply:
Done